### DISCUSSION TITLE
Theming with 9.0.0

### DISCUSSION THREAD

**Discussion by paulgrieselhuber** [Shopify/polaris-react#5227] [Q&A]
Since [themes appear to have been nixed](https://github.com/Shopify/polaris-react/pull/4642), and the docs are out of date (7.6.0), what is the guidance around theming now?

**Comment by aaronccasanova** [comment]
Guidance for theming can be found in the Polaris v9 [migration guide](https://github.com/Shopify/polaris-react/blob/a068ebf84a098cc95512814032e4f7afafb2ac11/documentation/guides/migrating-from-v8-to-v9.md#from-themeprovider-to-customproperties). However, it's important to note there is no obligation to upgrade to Polaris v9 at this time. We are currently working on the upgrade internally and plan to make a public announcement about the release in the near future.
